.gitignore

```diff
@@ -90,3 +90,4 @@ NuGetExeBuild/
 NuGetRefBuild/
 Packages/
 NuGetTaskBuild/
+build/
```

Comment:
@orjan Since those files are already committed, it shouldn't matter. But we can be explicit about it:

```
build/*
!build/Build.cmd
!build/Update-GitVersionTfsTaskVersion.ps1
```